packages/abi-gen-templates/partials/tx.handlebars

```diff
@@ -54,6 +67,12 @@ public {{languageSpecificName}} = {
         })(),
     );
     },
+    /**
+     * Estimate gas to send the transaction
```

Comment:
`Estimates the gas cost of sending an Ethereum transaction calling a specific method with specific arguments.`